test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -184,20 +184,31 @@ func (s *snapshottableTestSuite) DefineTests(driver storageframework.TestDriver,
 			nodeName := pod.Spec.NodeName
 			gomega.Expect(nodeName).NotTo(gomega.BeEmpty(), "pod.Spec.NodeName must not be empty")
 
-			ginkgo.By(fmt.Sprintf("[init] waiting until the node=%s is not using the volume=%s", nodeName, pv.Name))
+			// When checking a node's volumeInUse this can be one of two values:
+			// - the pv.Name for in-tree volumes
+			// - the VolumeHandle for CSI volumes
+			volumeName := ""
+			if pv.Spec.PersistentVolumeSource.CSI != nil {
+				volumeName = pv.Spec.PersistentVolumeSource.CSI.VolumeHandle
+			} else {
```

Comment:
snapshots only work for csi volumes

Reply:
I've updated this to always use the volumeHandle... but I'm curious. This was taken from the existing snippet that was ran in Windows environments. If we only run these tests for CSI Volumes, then it would imply that this condition has never worked?

Reply:
I'm not aware of it. I'm assuming it was never exercised.